src/Nethermind/Nethermind.JsonRpc.Test/Data/Eip2930Tests.cs

```diff
@@ -0,0 +1,209 @@
+//  Copyright (c) 2021 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+// 
+
+using System.Collections.Generic;
+using FluentAssertions;
+using Nethermind.Core;
+using Nethermind.Core.Eip2930;
+using Nethermind.Core.Test.Builders;
+using Nethermind.Int256;
+using Nethermind.JsonRpc.Data;
+using Nethermind.Serialization.Json;
+using NUnit.Framework;
+
+namespace Nethermind.JsonRpc.Test.Data
+{
+    public class Eip2930Tests
+    {
+        private readonly EthereumJsonSerializer _serializer = new EthereumJsonSerializer();
+        private Transaction _transaction = new Transaction();
+        private Dictionary<Address, IReadOnlySet<UInt256>> _data = new Dictionary<Address, IReadOnlySet<UInt256>>();
+        private TransactionForRpc _transactionForRpc = new TransactionForRpc();
+        private HashSet<UInt256> _storageKeys = new HashSet<UInt256>();
+
+        private AccessList GetTestAccessList()
+        {
```

Comment:
you should also test empty access list and account with no storage

Reply:
I'm testing it. This method is for testing standard, valid access list, maybe the name is a bit confusing, I was thinking about naming it like "GetTestValidAccessList", but finally I chose just "GetTestAccessList" to avoid name with to many adjectives. I moved it to separate method because this code is used 3 times, in 3 different tests and I wanted to avoid duplicating. Empty access list is tested in lines 101-113 in  test "can_serialize_empty_accessList" and as the code occurs only once, it's directly inside the test. Empty storage is tested in lines 124-139 in test "can_serialize_accessList_with_empty_storageKeys".